tox.ini

```diff
@@ -30,6 +30,11 @@ deps = flake8
        flake8-logging-format
 commands = flake8
 
+[testenv:readme]
+deps = 
+    collective.checkdocs
```

Comment:
How different it is than `doc8`?

Reply:
It just validate for possibility to render Readme on PyPI side. Nothing more.